tests/unit/packaging/test_models.py

```diff
@@ -668,6 +668,26 @@ def test_trusted_published_all(self, db_session):
 
         assert release.trusted_published
 
+    def test_is_url_verified(self, db_session):
```

Comment:
Usually better to break these up so it's one assertion per test. You can use `pytest.mark.parametrize` to run a test multiple times with different parameters. I would recommend something like:

```suggestion
  @pytest.mark.parametrize(
      "url, expected",
      [
          ("xpto.com", False),
          ("https://fake/", False),
          ("https://fake/url", True),
          ("https://fake/url/something.md", True),
      ]
  )
  def test_is_url_verified(self, db_session, url, expected):